install/linux/usr/share/odemis/sim/enzel-sim.odm.yaml

```diff
@@ -285,6 +286,17 @@ ENZEL-Sim: {
   # used.
 }
 
+# Special component to provide control of the 5DOF in a global reference frame
+"Global 5DOF Aligner": {
```

Comment:
I'm not very found of the name. Particularly because it implies is has 5 axes, but there are only 3. I'd suggest to name it just "Stage Global". You could also rename the "Stage Z Global" to "Stage YZ Global" to clarify further all these wrappers.